src/Automation/Interfaces/IOutputFileHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.Automation
+{
+    internal interface IOutputFileHelper
```

Comment:
>internal [](start = 4, length = 8)

Doesn't this interface need to be public, since it's part of the public API in SnapshotCommand?